### PR TITLE
(SIMP-3164) Update static assets

### PR DESCRIPTION
**Pull request by op-ct**
This commit normalizes static assets with other SIMP Puppet modules.
In particular, Travis CI is configured to include Puppet 4.10.

SIMP-3164 #comment Updated simp-simp_grafana

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
   gem 'metadata-json-lint'
+  gem 'puppet-strings'
   gem 'puppet-lint-empty_string-check',   :require => false
   gem 'puppet-lint-trailing_comma-check', :require => false
   gem 'simp-rspec-puppet-facts', ENV.fetch('SIMP_RSPEC_PUPPET_FACTS_VERSION', '~> 1.3')
-  gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', '~> 3.5')
-
-  # extra gems (defined in build/pupmod_metadata/extra_gems.yaml)
-  gem 'toml'
+  gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', '~> 3.0')
```
Comment (lnemsick-simp) [inline Gemfile]: Missing toml gem.

File: Gemfile [hunk 2]
```diff
@@ -39,5 +37,5 @@ end
 group :system_tests do
   gem 'beaker'
   gem 'beaker-rspec'
-  gem 'simp-beaker-helpers', ENV.fetch('SIMP_BEAKER_HELPERS_VERSION', '~> 1.7')
+  gem 'simp-beaker-helpers', ENV.fetch('SIMP_BEAKER_HELPERS_VERSION', '~> 1.5')
```
Comment (lnemsick-simp) [inline Gemfile]: why is this bumped down?

File: Gemfile (hunk 1 above)
Comment (lnemsick-simp) [inline Gemfile]: why is SIMP_RAKE_HELPERS_VERSION bumped down?

**Review by lnemsick-simp — CHANGES_REQUESTED** [review]

**Review by jeefberkey — APPROVED** [review]

**Review by lnemsick-simp — APPROVED** [review]

**Comment by lnemsick-simp** [comment]
Please don't merge until I run the acceptance tests.  Should not have accepted changes before I was done testing.

**Review by lnemsick-simp — APPROVED** [review]
Acceptance test fails, but appears to be independent of static assets update.

**Pull request merged**